### ISSUE TITLE
Playtest feature list!

### ISSUE THREAD

**Issue by cmavelis** [cmavelis/zug#7]
Checklist of what needs to be done before I'll subject loved ones to play testing:

- [x] Lobby system (deferred a _true_ lobby for now) #8 
- [x] basic UI for orders https://github.com/cmavelis/simulchess/pull/13
- [x] basic UI for turn resolution (optional) https://github.com/cmavelis/simulchess/pull/21
- [x] logic for resolving a "push" https://github.com/cmavelis/simulchess/pull/11 (optional) (but will make the game much more interesting)
  - Missing some edge cases
- [x] win condition (oh right)
  - won't do this yet.  Not sure how the endgame should work at this point. Will tell playtesters
